docs/self-hosted/quickstart.md

```diff
@@ -1,158 +1,64 @@
-</video>
+## Run Directus 
 
-We'll be calling our Field `title`. While Directus offers a range of powerful field customization options, we'll be
-sticking to the defaults for now. These defaults use the "String" datatype.
+Run the following in your terminal:
 
-::: tip More Info on Fields
-
-To learn more, see our documentation on [Fields](/getting-started/glossary#fields).
-
-:::
-
-## 5. Create an Item
-
-Now that we have a Collection with a Field configured, it's time to start adding some content. Navigate to the Content
-Module (top left), and click <span mi btn>add</span> in the top-right to get started. This will take you to the
-Create/Edit Item page:
-
-![Directus Create Item](https://cdn.directus.io/docs/v9/getting-started/quickstart/quickstart-20220217A/create-item-20220217A.webp)
-
-Once you're happy with your creation, click <span mi btn>check</span> in the top-right to save your Item to the
-database.
-
-::: tip More Info on Items
-
-To learn more, see our documentation on [Items](/app/content/items).
-
-:::
-
-## 6. Set Role/Public Permissions
-
-By default, all content entered into Directus is considered private. This means that no data will be returned by the
-API, unless requested by an authenticated user that has the correct permissions. In order to have the API return our
-items, we'll have to setup some permissions. Navigate to **Settings Module <span mi icon dark>chevron_right</span> Roles
-& Permissions**.
-
-Directus ships with a special **"Public"** role that controls what data is returned to non-authenticated users. Select
-the Public Role, find your Collection, and click the icon under the <span mi icon>visibility</span> icon (read/view
-permission) to allow the Public Role to read the Items in your Collection.
-
-![Directus Permissions](https://cdn.directus.io/docs/v9/getting-started/quickstart/quickstart-20220217A/permissions-20220217A.webp)
-
-::: tip More Info on Roles & Permissions
-
-Roles & Permissions are extremely powerful and can get pretty in-depth. To learn all about the nuances in setting these
-up, see [Roles](/reference/system/roles) & [Permissions](/reference/system/permissions).
-
-:::
-
-## 7. Connect to the API
-
-Now that your project has some content in it, it's time to start using this content externally. Data can be accessed in
-a number of ways, including the REST API, GraphQL, the CLI, or even straight from the database. In this case, we'll use
-[the `/items/` REST API endpoint](/reference/items) to retrieve the item we just created.
-
-Use your browser or an API tool like [Postman](http://postman.com) or [Paw](https://paw.cloud) to open
-`http://localhost:8055/items/articles`.
-
-And there it is! The Article Item you just created is being served in beautiful JSON, ready to be used anywhere and
-everywhere!
-
-```json
-{
-	"data": [
-		{
-			"id": 1,
-			"title": "Hello World!"
-		}
-	]
-}
 ```
+docker-compose up
```

Comment:
While i do agree `docker-compose up` may be easier for someone using docker for the very first time as you'll instantly see the logs but we probably should mention that `docker-compose up -d` is the best practice way of starting your dockers
```suggestion
docker compose up
```
with modern docker the compose command is now part of docker instead of a separate tool with a dash.

Reply:
Oh, annoying. I didn't know that and in the video it's with a dash. Is there a material difference? If not I may suggest leaving this for consistency

Reply:
With a dash is V1 and will not be removed for new versions after June 2023 😬 https://docs.docker.com/compose/reference/ (didnt know that deadline was now, i still use V1 all over the place)

Reply:
Yikes ok.

Reply:
Yeah, really annoying but I agree with @br41nslug that we should use `docker compose` everywhere